sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
```

Comment:
ou dont need this file here. it is enought if you test in the test file that 1+ 2 equals to 3